### DISCUSSION TITLE
Welcome to powerschool Discussions!

### DISCUSSION THREAD

**Discussion by cbini** [TEAMSchools/powerschool#28] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jeganchen** [comment]
could you please share an example on updating/inserting records in this repository
 https://github.com/TEAMSchools/powerschool

i try but get below errors.

 Message=<?xml version="1.0" encoding="UTF-8" standalone="yes"?><errorMessage><message>Unrecognized field "Passport_number" (class com.pearson.pss.sis.api.SchemaRecord), not marked as ignorable (6 known properties: "@extensions", "tables", "message", "_extension_data", "id", "name"])
 at [Source: (org.apache.cxf.transport.http.AbstractHTTPDestination$1); line: 1, column: 22] (through reference chain: com.pearson.pss.sis.api.SchemaRecord["Passport_number"])</message></errorMessage>
  Source=C:\Users\admin\Documents\powerschool\powerschool-3.6.0\powerschool-3.6.0\tests\test_powerschool.py
  StackTrace:

During handling of the above exception, another exception occurred:

  File "C:\Users\admin\Documents\powerschool\powerschool-3.6.0\powerschool-3.6.0\tests\test_powerschool.py", line 22, in <module> (Current frame)
    u_student_ext.update(pk = 10090, body = body)
